components/Common/Search/States/index.module.css

```diff
@@ -0,0 +1,215 @@
+.searchBoxModalContainer {
+  @apply fixed
+    inset-0
+    z-50
+    flex
+    items-center
+    justify-center
+    bg-neutral-900
+    bg-opacity-90
+    dark:bg-neutral-900
+    dark:bg-opacity-90;
+}
+
+.searchBoxModalPanel {
+  @apply fixed
+    h-screen
+    w-full
+    bg-neutral-100
+    p-2
+    dark:bg-neutral-950
+    md:top-60
+    md:h-[450px]
+    md:max-w-3xl
+    md:rounded-xl
+    md:shadow-lg;
+}
+
+.searchBoxInnerPanel {
+  @apply pt-12
```

Comment:
FYI that a lot of tokesn are still improperly indented. It's always 2 spaces more than the `@apply`

I believe at some point we should make a prettier plugin. @canerakdas :P

Reply:
Yeah, Prettier tries to format these CSS files, but doesn't really format correctly

Reply:
Yeah this spacing is something that prettier is unable to do out of the box